rust/rbac-registration/src/cardano/transaction/raw_aux_data.rs

```diff
@@ -41,19 +40,19 @@ impl RawAuxData {
         let mut decoder = Decoder::new(aux_data);
 
         match decoder.datatype() {
-            Ok(minicbor::data::Type::Map) => {
+            Ok(Type::Map) => {
```

Comment:
This file can be safely remove, since the logic is moved to `cardano-blockchain-types`